contrib/opencensus-ext-azure/README.rst

```diff
@@ -16,103 +16,165 @@ Installation
+    logger.addHandler(handler)
 
-This example shows how to send a warning level log to Azure Monitor.
+    tracer = Tracer(exporter=AzureExporter(), sampler=ProbabilitySampler(1.0))
+
+    logger.warning('Before the span')
+    with tracer.span(name='test'):
+        logger.warning('In the span')
+    logger.warning('After the span')
+
+Metrics
+~~~~~~~
+
+The **OpenCensus Azure Monitor Metrics Exporter** allows you to export metrics to `Azure Monitor`_.
 
 * Create an Azure Monitor resource and get the instrumentation key, more information can be found `here <https://docs.microsoft.com/azure/azure-monitor/app/create-new-resource>`_.
 * Put the instrumentation key in ``APPINSIGHTS_INSTRUMENTATIONKEY`` environment variable.
+* You can also specify the instrumentation key explicitly in the code, which will take priority over a set environment variable.
 
-.. code:: python
+Using the Metrics exporter
+*****************************
 
-    import logging
+    .. code:: python
 
-    from opencensus.ext.azure.log_exporter import AzureLogHandler
+        import time
 
-    logger = logging.getLogger(__name__)
-    logger.addHandler(AzureLogHandler())
-    logger.warning('Hello, World!')
+        from opencensus.ext.azure import metrics_exporter
+        from opencensus.stats import aggregation as aggregation_module
+        from opencensus.stats import measure as measure_module
+        from opencensus.stats import stats as stats_module
+        from opencensus.stats import view as view_module
+        from opencensus.tags import tag_map as tag_map_module
 
-You can enrich the logs with trace IDs and span IDs by using the `logging integration <../opencensus-ext-logging>`_.
+        stats = stats_module.stats
+        view_manager = stats.view_manager
+        stats_recorder = stats.stats_recorder
+
+        CARROTS_MEASURE = measure_module.MeasureInt("carrots",
+                                                    "number of carrots",
+                                                    "carrots")
+        CARROTS_VIEW = view_module.View("carrots_view",
+                                        "number of carrots",
+                                        [],
+                                        CARROTS_MEASURE,
+                                        aggregation_module.CountAggregation())
+
+
+        def main():
+            # Enable metrics
+            # Set the interval in seconds in which you want to send metrics
+            exporter = metrics_exporter.new_metrics_exporter(export_interval=2)
+            view_manager.register_exporter(exporter)
+
+            view_manager.register_view(CARROTS_VIEW)
+            mmap = stats_recorder.new_measurement_map()
+            tmap = tag_map_module.TagMap()
+
+            mmap.measure_int_put(CARROTS_MEASURE, 1000)
+            mmap.record(tmap)
+            time.sleep(10)
+
+            print("Done recording metrics")
+
+
```

Comment:
I understand that the code is copied from the examples, I wonder if two empty lines here in a doc would be an overkill?

Reply:
Sure. I can remove it.